### ISSUE TITLE
pipenv install raises error

### ISSUE THREAD

**Issue by EricSchles** [pypa/pipenv#1603]
Be sure to check the existing issues, both open and closed.

Describe the issue briefly here.

When I attempt to run `pipenv install -r requirements.txt` I get an error.

##### Describe your environment

1. OS Type: Max OSX
1. Python version: `$ python -V`: Python 3.6.4
1. Pipenv version: `$ pipenv --version`: pipenv, version 11.1.4

##### Expected result

Describe what you expected.

I expected to be able to install my packages.

##### Actual result

When possible, provide the verbose output (`--verbose`), especially for locking and dependencies resolving issues.

When I attempt to run `pipenv install -r requirements.txt` I get the following error:

```
Erics-MBP:server erics$ pipenv install -r requirements.txt
Requirements file provided! Importing into Pipfile…
Pipfile.lock not found, creating…
Locking [dev-packages] dependencies…
ibrary/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pipenv/patched/pip/index.py", line 31, in <module>
    from pip.wheel import Wheel, wheel_ext
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pipenv/patched/pip/wheel.py", line 6, in <module>
    import compileall
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/compileall.py", line 20, in <module>
    from concurrent.futures import ProcessPoolExecutor
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pipenv/../concurrent/futures/__init__.py", line 8, in <module>
    from concurrent.futures._base import (FIRST_COMPLETED,
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pipenv/../concurrent/futures/_base.py", line 381
    raise exception_type, self._exception, self._traceback
                        ^
SyntaxError: invalid syntax
```

##### Steps to replicate

Provide the steps to replicate (which usually at least includes the commands and the Pipfile).

```mkdir testing_pipenv
pipenv --three
pipenv install requests

I also tried:

pipenv install pandas
```

https://github.com/EricSchles/testing_pipenv is a test repo I created, just in case that's helpful.

**Comment by techalchemy** [comment]
@EricSchles we just cut a release which should resolve this issue, should be fixed by c7c0613 which is now released

**Comment by EricSchles** [comment]
@techalchemy Thanks!  I'll check it out.
